packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -208,6 +209,49 @@ cluster.addNodegroupCapacity('custom-node-group', {
 });
 ```
 
+#### Node Groups with IPv6 Support
+
+Node groups are available with IPv6 configured networks.  Additional permissions are necessary in order for pods to obtain an IPv6 address.
+
+> For more details visit [Configuring the Amazon VPC CNI plugin for Kubernetes to use IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/cni-iam-role.html#cni-iam-role-create-role)
+
+```ts
+const ipv6Management = new iam.PolicyDocument({
```

Comment:
Could we do this automatically for the user if the cluster was created with `ipFamily === 'ipv6'`?

Reply:
Possibly.  I think the real fix here is just getting the default CNI managed policy to add these permissions.  I'm struggling to think of why such a change would be rejected.

Reply:
I'm taking a look at making a change to be able to do this.  It'll increase the scope of the change, but unless we modify the default node group's role permissions to include ipv6 this will be a headache for everyone because default node groups will never succeed out of the box with this change.

A cheap(er) way to do it would be to just directly modify the nodeGroup role to always include ipv6, but I think having the ability to measure the cluster ipFamily through CDK has value, too.